install/README.md

```diff
@@ -213,6 +215,43 @@ A list of _datalogger_ sampling configurations for a given _station_ and recordi
 | _Start_ | Stream start time|
 | _Stop_ | Stream stop time|
 
+#### _GAINS_ ####
+ 
+A list of installation times where gains need to be applied to correct for local conditions, datalogger or sensor settings.
+For the scale factor and bias either a value can be given directly or an expression can be used if that makes it clearer where the number has come from.
+
+| Field | Description | Units |
+| --- | --- | --- |
+| _Station_ | Datalogger recording _Station_|
+| _Location_ | Recording sensor site _Location_ |
+| _SubLocation_ | Features _SubLocation_ if applicable |
+| _Channel_ | The sensor channel, as defined in the response configuration, which requires a gain adjustment, multiple channels can be joined (e.g _"Z"_ or _"ZNE"_).
+| _Scale Factor_ | Scale, or gain factor, that the input signal is multiplied by prior to digitisation, or for polynomial responses it is the factor used to convert Volts into the signal units. If this field is empty, it should be assumed to have a value of __1.0__ which in theory should have no impact.
+| _Scale Bias_ | An offset value that needs to be added to the signal prior to digitisation and indicates a polynomial response is expected, if this field is blank it is assumed that the value is __0.0__.
```

Comment:
those are also provided in the calibrations.
If in the gains we need to provide "site specific" scale factor and bias, then we might adjust the description accordingly?